packages/salesforcedx-sobjects-faux-generator/src/constants.ts

```diff
@@ -9,3 +9,5 @@ export const CLIENT_ID = 'sfdx-vscode';
 export const SFDX_DIR = '.sfdx';
 export const TOOLS_DIR = 'tools';
 export const SOBJECTS_DIR = 'sobjects';
+export const STANDARDOBJECTS_DIR = 'standardObjects';
```

Comment:
These changes shouldn't be part of this PR.

Reply:
Thanks, I didn't double check the base branch